checkov/common/runners/object_runner.py

```diff
@@ -103,6 +104,7 @@ def run(
             self.pbar.set_additional_data({'Current File Scanned': os.path.relpath(file_path, root_folder)})
             skipped_checks = collect_suppressions_for_context(definitions_raw[file_path])
             results = registry.scan(file_path, definitions[file_path], skipped_checks,runner_filter)  # type:ignore[arg-type] # this is overridden in the subclass
+            self.modify_gha_keys_in_results(results, definitions[file_path])
```

Comment:
@Eliran-Turgeman Can this be corrected in the scan itself, or only in hindisght?

Reply:
@nimrodkor moved the implementation into the scan